gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
-#Sat Nov 28 12:59:24 CET 2015
+#Mon Apr 30 17:32:05 CEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-bin.zip
-distributionSha256Sum=c58650c278d8cf0696cab65108ae3c8d95eea9c1938e0eb8b997095d5ca9a292
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```

Comment:
Why upgrading from `-bin.zip` to `-all.zip`?

Reply:
Android Studio automatic update. Do you prefer to keep `bin.zip`?

Reply:
Made the change